### PR TITLE
PodDeletionCost to Beta

### PR DESCRIPTION
**Pull request by ahg-g**
Ref: kubernetes/enhancements#2255

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ different Kubernetes components.
 | `NodeDisruptionExclusion` | `true` | Beta | 1.19 | 1.20 |
 | `NonPreemptingPriority` | `false` | Alpha | 1.15 | 1.18 |
 | `NonPreemptingPriority` | `true` | Beta | 1.19 | |
-| `PodDeletionCost` | `false` | Alpha | 1.21 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:149]: Please leave this line in place.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ different Kubernetes components.
...
+| `PodDeletionCost` | `true` | Beta | 1.21 | 1.22 |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:149]: I think you mean
```suggestion
| `PodDeletionCost` | `true` | Beta | 1.22 |   |
```

**Review by sftim — DISMISSED** [review]
Suggestions for correctness

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:149]: (the last column should change to `1.21`)

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 2 above)
Comment (ahg-g) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: oops, I clearly didn't look at the header :)
done.

**Comment by carlisia** [comment]
Hey @ahg-g, this lgtm but it needs a rebase please.

**Comment by ahg-g** [comment]
rebased

**Comment by ahg-g** [comment]
Friendly ping, can we please get this approved before it gets into another conflict?

**Comment by sftim** [comment]
Graduation PR https://github.com/kubernetes/kubernetes/pull/101080 has merged.

**Pull request merged**